theory.md

```diff
@@ -56,4 +56,8 @@ Good Luck
 1.	Explain the Git workflow
 git flow involves isolating your work into different types of Git branches. Git workflows encourage developers and devops teams to leverage Git effectively and consistently. Git offers a lot of flexibility in how users manage changes
 
+2.	How do you clone a repository? 
+git clone <repository url> [directory]
+3.	How do you check the status of your Git repository?  
+git status command 

```

Comment:
thanks. but can you please remove the trailing command at the end.

just "git status" is fine

Reply:
noted, will do.